modules/subscriber/variables.tf

```diff
@@ -104,3 +104,10 @@ variable "lambda_env_vars" {
   nullable    = false
   default     = {}
 }
+
+variable "debug_endpoint" {
+  description = "Endpoint to send debugging telemetry to. Ask your Observe POC for further instructions"
```

Comment:
Don't worry about pointing folks to an internal contact. Point out that this is a short hand for setting `OTEL_EXPORTER_OTLP_ENDPOINT` in the lambda.